staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -939,7 +939,8 @@ func (e *Store) updateForGracefulDeletionAndFinalizers(ctx context.Context, name
 			if !graceful {
 				// set the DeleteGracePeriods to 0 if the object has pendingFinalizers but not supporting graceful deletion
 				if pendingFinalizers {
-					klog.V(6).Infof("update the DeletionTimestamp to \"now\" and GracePeriodSeconds to 0 for object %s, because it has pending finalizers", name)
+					klog.V(6).InfoS("Having pending finalizers, then mark status to deleting", "obj", name,
```

Comment:
We should keep the subject consistent in the log message.

The subject in "Having pending finalizers" is the object, while the subject in "then mark status to deleting" is the registry.

I'm not sure how to rewrite this log message, @serathius can you help?

Maybe like this:

```golang
klog.V(6).Infof("Object has pending finalizers, so the registry is going to update it with", "object", "klog.KObj(obj), "DeletionTimestam", "\"now\"", "GracePeriodSeconds", 0)
```

Reply:
I got namespace from ctx , updated